op-acceptor/nat.go

```diff
@@ -85,6 +86,14 @@ func New(ctx context.Context, config *Config, version string, shutdownCallback f
 // Start runs the acceptance tests periodically at the configured interval.
 // Start implements the cliapp.Lifecycle interface.
 func (n *nat) Start(ctx context.Context) error {
+	// Set up panic recovery to ensure we exit with code 2 for runtime errors
+	defer func() {
+		if r := recover(); r != nil {
+			n.config.Log.Error("Runtime error occurred", "error", r)
+			os.Exit(exitcodes.RuntimeErr)
```

Comment:
We're not putting this in main.go because that's a different goroutine, correct?

Reply:
The fact that we have two different `os.Exit(exitcodes.RuntimeErr)` calls reveals the complexity introduced by doing the initial test run within `Start()`... not a huge deal though.

Reply:
Agreed.
Let me know if you have any ideas on reducing this complexity.